### PR TITLE
Copy edit the README.

### PR DESCRIPTION
**Pull request by jpmedley**
I was reading README in prep for an article that Pete's writing. Thought I'd fix things along the way.

I've added comments to explain the reasons for the suggestions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ This proposal does not address the source of the contacts. Many operating system
...
   * Excluding the operating system, there is no canonical place where users maintain a list of their contacts. Developers would have to integrate with a variety of services to provide a comprehensive experience.
-  * User agents can offer an experience to users that's consistent with other applications on the user's device. The information that will is going to be shared with the website can be clearly highlighted.
-  * Contact information is sensitive. Individual contacts can have [any number of properties](https://en.wikipedia.org/wiki/VCard#Properties) associated with them. Websites seeking the e-mail address of a single user should not have access to the same information of all other contacts.
+  * User agents can offer an experience to users that's consistent with other applications on the user's device. The information that will be shared with the website can be clearly highlighted.
+  * Contact information is sensitive. Individual contacts can have [any number of properties](https://en.wikipedia.org/wiki/VCard#Properties) associated with them. Websites seeking the e-mail address of a single user should not have access to all other contacts.
```
Comment (jpmedley) [inline README.md]: **Omit needless words** - If a word can be removed while the sentence retains its meaning, then the word didn't need to be there. If deletion changes the meaning, then said meaning was probably not properly conveyed and the sentence needs to be rewritten.

File: README.md [hunk 2]
```diff
@@ -56,22 +56,22 @@ partial interface Navigator {
 };
 ```
 
-  * Sequences are returned for the properties as multiple may be available in the user's address book. User agents are encouraged to enable users to limit their selection.
-  * Support for additional properties can be added iteratively. Whether the returned data can (and should) be sanitized is a question that's unique to each property. At least one property must be provided.
+  * Sequences are returned for the properties as multiple values may be available in the user's address book. User agents are encouraged to enable users to limit their selection.
+  * Support for additional properties can be added iteratively. Whether the returned data can (and should) be sanitized is a question that's unique to each property. The user agent must provide at least one property.
   * Some future might include the ability to add contacts, or even _contact management_, so having an intermediary object on `navigator` helps extensibility.
```
Comment (jpmedley) [inline README.md]: It's unclear who has to provide the one property. The sentence was also in passive voice.

File: README.md [hunk 3]
```diff
@@ -56,22 +56,22 @@ partial interface Navigator {
...
 
 ## Security and Privacy
-Exposing contact information has a clear privacy impact. We propose a picker model so that the user agent can offer a user experience that makes it clear what information in going to be shared with the website: the spec will have a MUST requirement to ensure that the user understands and selects which contacts or information to share.
+Exposing contact information has a clear privacy impact. We propose a picker model so that the user agent can make it clear what information in going to be shared with the website; the spec will have a MUST requirement to ensure that the user understands and selects which contacts or information to share.
```
Comment (jpmedley) [inline README.md]: Omit needless words.

**Comment by rayankans** [comment]
Thanks for improving the explainer!

**Pull request merged**